files/base.rb

```diff
@@ -63,6 +63,24 @@ def team_data(lookup_key = nil)
     data
   end
 
+  def check_data(lookup_key = nil)
```

Comment:
if someone does want the over engineered ruby like you previously had I was going to refactor these three to use something common.